aemedge/scripts/article.js

```diff
@@ -42,19 +42,47 @@ async function allAuthorEntries(articleStream) {
   return getAuthorEntries(Array.from(authorSet));
 }
 
+function buildAuthorUrl(author) {
+  return `/author/${toClassName(author.trim()).replaceAll('-', '')}`;
+}
+
 function asEntry(authorName) {
   return (authorName === '0')
     ? null
-    : { author: authorName, path: `/author/${toClassName(authorName).replace('-', '')}` };
+    : { author: authorName, path: buildAuthorUrl(authorName) };
 }
 
 /* AuthorEntry for the given article, or else fallback */
 function authorEntry(article, authorEntries) {
   return authorEntries.find((e) => e.author === article.author) || asEntry(article.author);
 }
 
+const defaultSuffixes = ['PhD', 'Ph.D.'];
+function removeAuthorsSuffixes(authors, suffixes = defaultSuffixes) {
+  if (!authors) {
+    return '';
+  }
+  let authorsWithoutSuffixes = authors;
+  suffixes.forEach((suffix) => {
+    /**
+     * Jane Doe, PhD,
```

Comment:
I am not Markus Haack. If I was, I would assume he'd say: these comments have to go.